ncloud/resource_ncloud_subnet_test.go

```diff
@@ -0,0 +1,111 @@
+package ncloud
+
+import (
+	"fmt"
+	"regexp"
+	"testing"
+
+	"github.com/NaverCloudPlatform/ncloud-sdk-go-v2/services/vpc"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+)
+
+func TestAccResourceNcloudSubnet_basic(t *testing.T) {
+	var subnet vpc.Subnet
+	name := "testacc-subnet-basic"
+	cidr := fmt.Sprintf("10.2.2.0/24")
```

Comment:
```suggestion
	cidr := "10.2.2.0/24"
```
`fmt.Sprintf` 를 제거해도 되지 않나요?